fabric-client-tags-api-v1/src/client/resources/fabric.mod.json

```diff
@@ -0,0 +1,26 @@
+{
+  "schemaVersion": 1,
+  "id": "fabric-client-tags-api-v1",
+  "name": "Fabric Client Tags",
+  "version": "${version}",
+  "environment": "client",
+  "license": "Apache-2.0",
+  "icon": "assets/fabric-client-tags-api-v1/icon.png",
+  "contact": {
+    "homepage": "https://fabricmc.net",
+    "irc": "irc://irc.esper.net:6667/fabric",
+    "issues": "https://github.com/FabricMC/fabric/issues",
+    "sources": "https://github.com/FabricMC/fabric"
+  },
+  "authors": [
+    "FabricMC"
+  ],
+  "depends": {
+    "fabricloader": ">=0.4.0",
```

Comment:
Should be the version we are compling against, so `0.14.6`

Reply:
No, other modules cannot just be updated blindly. The version of loader denotes the mixin compat level for the mod, not that makes any difference here, just something we want to keep ontop of with new modules 👍